Lesson7/deniz/style.css

```diff
@@ -0,0 +1,370 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+
+:root {
+    --white: #ffffff;
+    --black: #000000;
+    --grey-lighter: #555;
+    --black-darker: #242424;
+    --white-darker: #F2F2F2;
+    --green:  #9acd32;
+}
+
+
+main {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: var(--white);
+    
+}
+
+.container-top {
+  width: 100%;
+}
+
+
+.h1-top {
+    padding-left: 2rem;
```

Comment:
If you want to use rem, you need to define font-size for :root, html or body element. Since there are many places where you use px, consider using px here instead